Scala/Scala.sublime-syntax

```diff
@@ -249,16 +368,26 @@ contexts:
                 3: variable.import.renamed-to.scala
             - match: '([^\s.,}]+)'
               scope: variable.import.scala
+
   inheritance:
     - match: '(extends|with)\s+([^\s\{\(\[\]]+)'
       captures:
         1: keyword.declaration.scala
         2: entity.other.inherited-class.scala
+    - match: '(extends|with)\s+\('
+      captures:
+        1: keyword.declaration.scala
+      push:
+        - match: '\)'
+          pop: true
+        - include: delimited-type-expression
+
   initialization:
-    - match: '\b(new)\s+({{id}})'
+    - match: '\b(new)\s+'
```

Comment:
I think `late-keywords` already takes care of that, but it's reasonable to do it here.